README.md

```diff
@@ -50,6 +50,18 @@ git clone https://github.com/kaitai-io/kaitai_struct_visualizer
 ksv <binary-file> <ksy-file>... | <rb-file>
 ```
 
+### Using Docker image
```

Comment:
By using the Docker image I mean the whole cycle - not only pull and run, but also build and run, and maybe push. Pushing requires local tag to match with remote repo.

Reply:
> By using the Docker image I mean the whole cycle - not only pull and run, but also build and run, and maybe push. Pushing requires local tag to match with remote repo.

Well, I get your point, but I wouldn't arrange the README like that. This section _Using Docker image_ is under a higher-level heading [_Usage_](https://github.com/kaitai-io/kaitai_struct_visualizer/blob/324cce7d05421a38f9bd800c26c549c8b099eacf/README.md#usage), so it should be only about the usage from the user's perspective, not about development. This command

```shell
docker run -v "$(pwd):/share":Z -it kaitai/ksv <binary-file> <ksy-file>
```

might be useful for hundreds of users (including the ones that perhaps haven't heard about Docker before) on a daily basis, whereas this one

```shell
docker build . --tag docker.io/kaitai/ksv
```

will only be used by a handful of developers (maybe) and only like once every 2 months. Moreover, there is nothing surprising on this `docker build` invocation, it is no different from the daily routine that Docker users are used to. So there is little point in even stating it in the README, and if you really insist, please don't put it into _Usage_, because it's not about daily usage, and rather put it into a _Development_ section or something like that.